ci-operator/config/opendatahub-io/ai-edge/opendatahub-io-ai-edge-main.yaml

```diff
@@ -18,6 +18,7 @@ resources:
       memory: 200Mi
 tests:
 - as: test-ai-edge
+  skip_if_only_changed: \.github|\.md$|^LICENSE$|^OWNERS$|THIRD-PARTY-NOTICES.txt|^acm/|^gitea/|pipelines/model-upload|pipelines/models|pipelines/tekton/gitops-update-pipeline|test/acm|test/gitops
```

Comment:
@MarianMacik Correct me if I'm wrong but this should only execute on changes that match this regex AND include changes to files that do not match like `pipelines/tekton/ai-edge`  and `test/e2e-tests` folder, correct?